lib/css/base/_stacks-configuration-dynamic.less

```diff
@@ -107,16 +107,6 @@
     @button-primary-number-color:                    var(--theme-secondary-900);
 
     //  Tags
-    @tags-border:                                    transparent;
-    @tags-background:                                var(--powder-100);
-    @tags-color:                                     var(--powder-700);
-    @tags-hover-border:                              transparent;
-    @tags-hover-background:                          var(--powder-200);
-    @tags-hover-color:                               var(--powder-800);
-    @tags-selected-border:                           transparent;
-    @tags-selected-background:                       var(--powder-400);
-    @tags-selected-color:                            var(--powder-900);
-
```

Comment:
This may be a premature optimization—we may need this for current Less-based theming. We'll have to test.